### DISCUSSION TITLE
Advice on NVENC AV1 encoding

### DISCUSSION THREAD

**Discussion by Neo166** [mafiosnik777/enhancr#29] [General]
Hi Team,
Appreciate your advice on the best optimum settings for NVENC AV1 encoding from size\quality trade off. I know NVENC is not very good but I got a 4090 and the CPU encodes take up to 2 hours on 3600x (yeah that's some old CPU with a powerful GPU ;-) ).

I currently use the default NVENC string of the enhancr except that i change the "-c:qv" value to 40.

-c:v av1_nvenc -preset p7 -tune:v hq -rc:v vbr -cq:v 40 -b:v 250M -pix_fmt nv12

Regards

**Comment by mafiosnik777** [comment]
Those default AV1 settings I have in enhancr suck, they were still from when ffmpeg git master just added av1_nvenc and was buggy as hell and these params were the only thing that worked without being broken.

Didn't bother to change since I don't hardware encode and also have no 40 series GPU.

I think changing to cq to 40 doesn't even do shit because of the fixed bitrate.
